hs_core/models.py

```diff
@@ -1119,6 +1119,7 @@ def bag_url(cls, resource_id):
     def delete(self, using=None):
         from hydroshare import hs_bagit
         from hs_access_control.models import UserResourcePrivilege, GroupResourcePrivilege
+        from hs_labels.models import UserResourceLabels
```

Comment:
I'm not insisting these *must* be removed and some times a local import here or there is a good solution when its the only place in a module that something is used anyway. I'm okay with leaving this, just outlining another option for future reference.